docs/exportJson.adoc

```diff
@@ -0,0 +1,109 @@
+Make sure to set the config options in your `neo4j.conf`
+
+.Neo4j.conf
+
+----
+apoc.export.file.enabled=true
+----
+
+// tag::export.json[]
+`YIELD file, source, format, nodes, relationships, properties, time, rows`
+[cols="1m,5"]
+|===
+| apoc.export.json.query(query,file,config) | exports results from the Cypher statement as Json to the provided file
+| apoc.export.json.all(file,config) | exports whole database as Json to the provided file
+| apoc.export.json.data(nodes,rels,file,config) | exports given nodes and relationships as Json to the provided file
+| apoc.export.json.graph(graph,file,config) | exports given graph object as Json to the provided file
+|===
+// end::export.json[]
+
+.Config
+
+|===
+| useTypes | true/false, if true export datatypes too.
+|===
+
+
+== Examples
+
+For the examples below we create the following dataset:
+
+[source,cypher]
+----
+CREATE (f:User {name:'foo',age:42,male:true,kids:['a','b','c']})-[:KNOWS]->(b:User {name:'bar',age:42}),(c:User {age:12})
+----
+
+.Export query
+
+[source,cypher]
+----
+CALL apoc.export.json.query("MATCH (u:User) WHERE age > {age} return u","/tmp/query.json",{params:{age:10}})
+----
+
+result:
+
+[source,json]
+----
+{"data":{"age":42,"name":"foo","male":true,"kids":["a","b","c"],"labels":["User"]}}

```

Comment:
this is not corrct, it should be a json object with one field per column of the query

Reply:
also the structure of the data is wrong, it should be the same for graph entities as the usual export format